src/components/MessagesList/MessagesList.vue

```diff
@@ -267,6 +267,9 @@ export default {
 			this.$store.dispatch('easeMessageList', { token: oldToken })
 			this.stopFetchingOldMessages = false
 			this.messagesGroupedByDateByAuthor = this.prepareMessagesGroups(this.messagesList)
```

Comment:
Another thing is that we prepareMessagesGroup separately in `token` watcher and `messagesList` watcher. Maybe move it from here, and adjust if block?
```js
// token watcher will handle the conversations change
if (oldMessages?.length && newMessages.length && newMessages[0].token !== oldMessages?.at(0)?.token) {
	// prepare new groups and replace instead
+	this.messagesGroupedByDateByAuthor = this.prepareMessagesGroups(newMessages)
	// also do not skip scrollToBottom, and computation of isScrollable, when switching between loaded chats
-	return
}
```